guides/dev/tests/README.md

```diff
@@ -140,6 +140,47 @@ Please note that some CKEditor plugins are needed for reasons that might not be
 
 If the editor behaves differently when testing and during development, try to add all plugins you use during the development (you can find such list in the [`config.js`](https://github.com/ckeditor/ckeditor-dev/blob/master/config.js) file) and then remove redundant ones. Please note that adding all existing CKEditor plugins might not be a good solution since, for example, the [BBCode plugin](http://ckeditor.com/addon/bbcode) will strip HTML in your output.
 
+### Tagging Tests
+
+Each test should be properly tagged, so it is easy to group and filter them while running. Tags should be added using `bender-tags` meta comment, like:
+
+    /* bender-tags: editor */
+    // Rest of the file...
+
+In case of unit tests `bender-tags` meta comment should be placed in `*.js` file and in `*.md` file for manual tests.
+
+**Common tags used in tests**:
+
+* bug/feature - test covers bug fix or a new feature. If test suit contains more general tests this tag should be omitted,
+* &lt;CKEditor version&gt; - CKEditor target version,
+* &lt;number&gt; - reference to GitHub issue,
+* trac&lt;number&gt; - reference to Trac issue,
+* custom - if test covers some specific area of the editor, additional tags can be use (like `word`, `selection`, etc).
+
+**For manual tests only**:
+
+* tc - test case, required to filter out generic tests (it is usually used together with &lt;CKEditor version&gt; tag so it is possible to test changes introduced in a specific version/release).
```

Comment:
We want to get rid of tc tag, see:

> `tc` - test case, required to filter out generic tests
> 
> Hm, that’s interesting remark… I haven’t thought about it. AFAIR the proposition was to completely ditch `tc` in favor of new `bug`/`feature`?

https://cksource.slack.com/files/msamsel/F5SR6QQDQ/Code_tagging

Let's simply skip it for now, and create a follow-up task to remove the `tc` tags.